src/openaq/incremental_downloader_parallel.py

```
@@ -418,6 +430,10 @@ def download_country_all(self, country_code: str, country_id: int,
                           f"Total: {total_measurements:,} measurements | ETA: {eta:.1f} min")
         
         else:
+            print(f"  → Using SEQUENTIAL location processing (parallel sensors within each location)")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)